lonboard-viz.ipynb

```diff
@@ -5,7 +5,23 @@
    "id": "56a301bb-74ef-4275-9c3d-419832f38297",
```

Comment:
<p>I'm confused by the labels on the color ramp. Can we fix those. Also can you put the notebook sharing links in the Readme at the top of the repo for easy access.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>